clang/include/clang/Analysis/FlowSensitive/DataflowAnalysisContext.h

```diff
@@ -209,6 +221,9 @@ class DataflowAnalysisContext {
     using DenseMapInfo::isEqual;
   };
 
+  DataflowAnalysisContext(Solver &S, std::unique_ptr<Solver> OwnedSolver,
```

Comment:
Please add comments either here or on the fields relating S and OwnedSolver.

Reply:
Done.